packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -108,6 +108,29 @@ class _Compiler {
         .createTempSync('output_dill');
     final File outputDill = outputDillDirectory.childFile('output.dill');
 
+    bool suppressOutput = false;
+    void reportCompilerMessage(String message) {
+      if (suppressOutput)
+        return;
+
+      if (message.startsWith('compiler message: Error: Could not resolve the package \'test\'')) {
+        printTrace(message);
+        printError('\n\nFailed to load test harness. Are you missing a dependency on flutter_test?\n');
+        suppressOutput = true;
+        return;
+      }
+
+      printError('$message');
+    }
+
+    ResidentCompiler createCompiler() {
+      return new ResidentCompiler(
+          artifacts.getArtifactPath(Artifact.flutterPatchedSdkPath),
+          packagesPath: PackageMap.globalPackagesPath,
+          trackWidgetCreation: trackWidgetCreation,
+          compilerMessageConsumer: reportCompilerMessage);
```

Comment:
please use flutter style for indentation of arguments (matches dartfmt's style with a trailing comma)